expandybird/Makefile

```diff
@@ -1,17 +1,22 @@
-# Makefile for the Docker image gcr.io/$(PROJECT)/expandybird
+# Makefile for the Docker image $(DOCKER_REGISTRY)/$(PROJECT)/expandybird
 # MAINTAINER: Jack Greenfield <jackgr@google.com>
 # If you update this image please check the tag value before pushing.
 
 .PHONY : all build test push container clean
 
-PREFIX := gcr.io/$(PROJECT)
+DOCKER_REGISTRY := gcr.io
```

Comment:
Should this be an env variable like $(PROJECT) so that when you want to push to different registries, you don't have to modify each file to override it?